go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/go-ldap/ldap
+module github.com/Zduygu/ldap
```

Comment:
Please revert this to the original name - I understand that this was for testing